undertow/src/main/java/org/wildfly/extension/undertow/ListenerResourceDefinition.java

```diff
@@ -76,6 +77,12 @@
             //.addDynamicRequirements(Capabilities.CAPABILITY_SERVER) -- has no function so don't use it
             .setAllowMultipleRegistrations(true) //hack to support mod_cluster's legacy profiles
             .build();
+
+    // only used by the subclasses Http(s)ListenerResourceDefinition
+    protected static final RuntimeCapability<Void> HTTP_UPGRADE_REGISTRY_CAPABILITY = RuntimeCapability.Builder.of(Capabilities.CAPABILITY_HTTP_UPGRADE_REGISTRY, true, ChannelUpgradeHandler.class)
```

Comment:
I believe the declaration of HttpListenerResourceDefinition.HTTP_UPGRADE_REGISTRY_CAPABILITY now needs to be removed as this constant takes its place.

Reply:
Yes, I think in an earlier version, I did remove it, but it somehow came back to life. Done.